src/server/test/web/readingsBarMeterFlow.js

```diff
@@ -7,307 +7,417 @@
     See: https://github.com/OpenEnergyDashboard/DesignDocs/blob/main/testing/testing.md for information.
 */
 
-const { chai, mocha, app } = require('../common');
-const Unit = require('../../models/Unit');
-const { prepareTest,
-    parseExpectedCsv,
-    createTimeString,
-    expectReadingToEqualExpected,
-    getUnitId,
-    ETERNITY,
-    METER_ID,
-    //unitDatakWh,
-    //conversionDatakWh,
-    //meterDatakWh 
-} = require('../../util/readingsUtils');
+const { chai, mocha, app } = require("../common");
```

Comment:
OED uses single quotes for strings throughout the code base unless double quotes are required, e.g., in nested strings. All the changes to double quotes need to be undone and also changed in your new code. I do see that some of the lines previously had double quote (I missed that before) but they should not have had that.